src/clr-angular/layout/tabs/_tabs.clarity.scss

```diff
@@ -98,8 +98,3 @@ button.nav-link {
     }
   }
 }
-
```

Comment:
Does it still work fine in FF after this removal?

Reply:
I tested in Firefox, no issues.